docs-src/cloud/api-keys/manage-api-keys.md

```diff
@@ -0,0 +1,122 @@
+---
+id: manage-api-keys
+title: Manage API Keys
+sidebar_label: Manage API Keys
+description: Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+tags:
+  - how to
+---
+
+Manage API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
+
+### Generate an API Key
+
+:::note
+
+Once generated, copy and securely save the API Key.
+It will be displayed only once for security purposes.
+
+:::
+
+Generate an API Key using one of the following methods.
+
+#### Using the Cloud UI
+
+To generate an API Key using the Cloud UI:
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
+2. Select **Settings** and choose **API Keys**.
+3. Select **Create API Key** and provide the following:
+   1. Name
+   2. Description
+   3. Expiration Date
+4. Select **Generate API Key**.
+
+:::note
+
+Temporal supports up to a 90-day duration for API Keys.
+
+:::
+
+#### Using the tcld
+
+To generate an API Key using tcld, use the `tcld apikey create` command.
+
+1. Authenticate your Temporal Cloud account using the [login](/cloud/tcld/login) command.
+2. Create an API Key using the [apikey create](/cloud/tcld/apikey/create) command.
+
+For example:
+
+```command
+tcld login
+tcld apikey create --name Your-Key-Name --description YourDescription --duration 24h
+```
+
+:::note
+
+Use tcld to create API Keys with a day level expiration by day.
+
+:::
+
+### Delete an API Key
+
+Delete an API Key using one of the following methods.
+
+#### Using the Cloud UI
+
+To delete an API Key using the Cloud UI
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
```

Comment:
Same feedback as above for Adding an API Key.  Please use the user guidance [here](https://www.notion.so/temporalio/Docs-for-API-Keys-78085708c5d64edd894dbb69ea9b82a5?pvs=4#d0ae56ad49064387ac9fbbbe237264cb) as only Global Admins can access Settings. 

To delete an API Key using the Cloud UI

1. [[Login](https://cloud.temporal.io/)](https://cloud.temporal.io/) to the Cloud UI
2. Navigate to your [[Profile Page → API Keys](https://cloud.temporal.io/profile/api-keys)](https://cloud.temporal.io/profile/api-keys)
3. Expand the vertical ellipsis to the right of the API Key’s row
4. Click Delete